_projects/email-bridge.md

```diff
@@ -0,0 +1,60 @@
+---
+name: First-Class Email Bridge
+desc: Build a "gateway" email bridge to Matrix
+requirements:
+# Student requirements:
+ - Keen interest in bridging networks together
+difficulty: Hard
+issues:
+mentors:
+# First person in contact; mentors may change before project starts.
+ - Half-Shot
+initiatives:
+ - GSoC
+tags:
+ - Node.JS
+ - Typescript
+ - Bridges
+ - Email
+ - SMTP
+---
+
+Email is one of the most ubiqutous methods on the internet to communicate today, having
+been around for a very long time. It's been a dream for a long time that one day Matrix
+could be connected up to Email by translating incoming [SMTP](https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol)
```

Comment:
uff. a first-class e-mail server?
Without having looked to much into it, I guess that an intermediate e-mail server & IMAP/POP3 is favourable for
- allowing access via other e-mail clients/tool
- all the spam-detection, categorization and security-related bits that e-mail servers do.

EDIT: Read the lines below and I see that this is exactly what you're looking for.